utils/build/docker/golang/chi.Dockerfile

```
@@ -1,17 +1,17 @@
-FROM golang:1.24 AS build
+FROM golang:1.24-alpine3.22 AS build
```

@nccatoni Provided that we were using `golang:1.24` that didn't pin against a specific `trixie` version, I'd suggest to do the same with this:

```suggestion
FROM golang:1.24-alpine AS build
```